src/napari/components/viewer_model.py

```diff
@@ -222,6 +222,7 @@ def __init__(
         super().__init__(
             title=title,
             dims={
+                'ndim': 2 if not len(axis_labels) else len(axis_labels),
```

Comment:
Re-reading, perhaps this should actually be
```suggestion
                'ndim': max(2, len(axis_labels))
```
to catch the case where the user only passes one axis label, given I think `ndim` always has to be a minimum of 2?

Reply:
hmm, with a single item in axis_labels how do we know which axis to label? 🤔